<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QFrame" name="frm_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>320</width>
      <height>680</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frm_next">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>120</width>
      <height>140</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_next">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Next:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>640</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start game</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_score_text">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_score">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>help</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_max_score_text">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Max_Score</string>
    </property>
   </widget>
   <widget class="QLabel" name="max1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
